src/operation/select-rule-comparison.bench.ts

```
@@ -0,0 +1,302 @@
+    { fixable: true, pattern: /^typescript-rule/ },
+    { fixable: false, pattern: /^react-rule/ },
+    { fixable: true, pattern: /^import-rule/ },
+    { fixable: true, pattern: /^style-rule/ },
+    { fixable: false, pattern: /^complexity-rule/ },
+    { fixable: false, pattern: /^security-rule/ },
+  ];
+
+  for (let index = 0; index < 10; index++) {
+    for (const { fixable, pattern } of rulePatterns) {
+      const ruleId =
+        pattern.source.replace("^", "").replace("$", "") + `-${index}`;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)